src/components/popover/react/Popover.tsx

```diff
@@ -40,54 +34,44 @@ enum PopperPlacement {
     LEFT_START = 'left-start',
 }
 
-interface IPopperOffsets {
+/**
+ * Vertical and horizontal offset of the popover.
+ */
+interface IOffsets {
     vertical?: number;
     horizontal?: number;
 }
-type PopperOffsets = IPopperOffsets;
+type Offsets = IOffsets;
```

Comment:
The plural is not necessary here I think. `IOffset` and `Offset` would be better I think

Reply:
Done